nimbus.fml.yaml

```diff
@@ -0,0 +1,72 @@
+---
+channels:
+  - release
+  - beta
+  - nightly
+  - developer
+features:
+  homescreen:
+    description: The homescreen that the user goes to when they press home or new tab.
+    variables:
+      sections-enabled:
+        description: "This property provides a lookup table of whether or not the given section should be enabled.
+        If the section is enabled, it should be toggleable in the settings screen, and on by default."
+        type: Map<HomeScreenSection, Boolean>
+        default:
+          {
+            "top-sites": true,
+            "jump-back-in": false,
+            "recently-saved": false,
+            "recent-explorations": false,
+            "pocket": false
+          }
+    defaults:
+      - channel: nightly
+        value: {
+          "sections-enabled": {
+            "top-sites": true,
+            "jump-back-in": true,
+            "recently-saved": true,
+            "recent-explorations": true,
+            "pocket": true
+          }
+        }
+  nimbus-validation:
+    description: "A feature that does not correspond to an application feature suitable for showing
+    that Nimbus is working."
+    variables:
+      settings-title:
+        description: The title of displayed in the Settings screen and app menu.
+        type: Option<String>
+        default: null
+      settings-punctuation:
+        description: The emoji displayed in the Settings screen title.
+        type: String
+        default: ""
+      settings-icon:
+        description: The drawable displayed in the app menu for Settings
+        type: String
+        default: mozac_ic_settings
+  search-term-groups:
+    description: A feature allowing the grouping of URLs around the search term that it came from.
+    variables:
+      enabled:
+        description: If true, the feature shows up on the homescreen and on the new tab screen.
+        type: Boolean
+        default: false
+types:
+  objects: {}
+  enums:
+    HomeScreenSection:
+        description: The identifiers for the sections of the homescreen.
+        variants:
+          top-sites:
+            description: The frecency and pinned sites.
+          recently-saved:
+            description: The sites the user has bookmarked recently.
+          jump-back-in:
+            description: The tabs the user was looking immediately before being interrupted.
+          recent-explorations:
+            description: The tab groups
+          pocket:
+            description: The pocket section. This should only be available in the US.
```

Comment:
This file format is documented in the [RFC for Nimbus-FML][1].

It's not going to move, but it's part of this PR so as to have something to feed into the `nimbus-fml`.

[1]: https://github.com/mozilla/experimenter-docs/pull/156